app/src/main/java/com/trianguloy/urlchecker/modules/companions/ShareUtility.java

```diff
@@ -75,19 +83,52 @@ public void onInitialize(View views) {
 
         /** Shares the url as text */
         public void shareUrl() {
```

Comment:
I understand this change, but for now I prefer to leave the share as just 'call android', without extra logic. It may change in the future though, but it will require other functionality present in the open that is still not here (like reordering)